src/com/cinergix/mapper/ObjectMapper.java

```diff
@@ -35,12 +47,179 @@
+	 * from <code>ResultSet</code>.
+	 * @param result Queried result set using a JDBC connection
+	 * @param columnName Label of the column which the value to be converted
+	 * @param typeClass Type of Class which the value should be convert to.
+	 * @return returns a converted value.
+	 * @throws SQLException
+	 */
+	protected Object parseValue( ResultSet result, String columnName, Class typeClass ) {
+		try{
+			
+			if( typeClass.equals( String.class ) ){
+				
+				return result.getString( columnName );
+			} else if( typeClass.equals( Byte.class ) || typeClass.equals( byte.class ) ){
+				
+				return result.getByte( columnName );
+			} else if( typeClass.equals( Short.class ) || typeClass.equals( short.class ) ){
+				
+				return result.getShort( columnName );
+			} else if( typeClass.equals( Integer.class ) || typeClass.equals( int.class ) ){
+				
+				return result.getInt( columnName );
+			} else if( typeClass.equals( Long.class ) || typeClass.equals( long.class ) ){
+				
+				return result.getLong( columnName );
+			} else if( typeClass.equals( Double.class ) || typeClass.equals( double.class ) ){
+				
+				return result.getDouble( columnName );
+			} else if( typeClass.equals( Float.class ) || typeClass.equals( float.class ) ){
+				
+				return result.getFloat( columnName );
+			} else if( typeClass.equals( Boolean.class ) || typeClass.equals( boolean.class ) ){
+				
+				return result.getBoolean( columnName );
+			} else if( typeClass.equals( Date.class ) ){
+				
+				return result.getTimestamp( columnName );
+			}
+			
+		}catch( SQLException ex ){
+			throw new DataTypeConversionException( "Unable to convert the value in " + columnName + " as " + typeClass.getName(), ex );
+		}
+		
+		return null;
+	}
+	
+	/**
+	 * To generate a Field/Column-label pair from <code>ResultField</code> annotation of properties ofGiven Class type 
+	 * @param dataClass
+	 * @return 
+	 */
+	protected HashMap<Field, String> getFieldColumnMapping( Class dataClass, ResultSet result ){
+		
+		if( dataClass == null || result == null ){
+			return null;
+		}
+		
+		HashMap<Field, String> map = new HashMap<Field, String>();
+
+		for( Field field : dataClass.getDeclaredFields() ){
+			
+			ResultField resultAnnotation = field.getAnnotation( ResultField.class );
+			if( resultAnnotation != null ){
+				
+				String sqlFieldName = resultAnnotation.value();
+				
+				if( checkColumnLabelExist( result, sqlFieldName ) ){
+					map.put( field, sqlFieldName );
+				}
+			}
+		}
+			
+		return map;
+	}
+	
+	protected boolean checkColumnLabelExist( ResultSet result, String columnLabel ){
+		
+		if( result == null || columnLabel == null || columnLabel.trim() =="" ){
+			return false;
+		}
+		
+		try{
+			
+			return result.findColumn( columnLabel ) > 0;
+		}catch( SQLException ex ){
+			// This exception thrown only when there is no matching column found in the result set.
```

Comment:
Only ignore the exception for expected error code. Other cases must throw the exception or throw a sensible exception.